rllib/examples/learner/train_w_bc_finetune_w_ppo.py

```diff
@@ -0,0 +1,162 @@
+import gymnasium as gym
+import shutil
+import tempfile
+import torch
+from typing import Mapping
+
+import ray
+from ray import tune
+from ray.air import RunConfig, FailureConfig
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import PPOTorchRLModule
+from ray.rllib.algorithms.ppo.ppo_catalog import PPOCatalog
+from ray.rllib.core.models.base import ACTOR, ENCODER_OUT
+from ray.rllib.core.rl_module.rl_module import SingleAgentRLModuleSpec
+from ray.rllib.examples.datasets.dataset_utils import convert_json_sample_batch_to_df
+
+GYM_ENV_NAME = "CartPole-v1"
+GYM_ENV = gym.make(GYM_ENV_NAME)
+
+
+class BCActor(torch.nn.Module):
+    """A wrapper for the encoder and policy networks of a PPORLModule.
+
+    Args:
+        encoder_network: The encoder network of the PPORLModule.
+        policy_network: The policy network of the PPORLModule.
+        distribution_cls: The distribution class to construct with the logits outputed
+            by the policy network.
+
```

Comment:
the args of init are always documented above init, not actually in init

Reply:
I've seen this somewhere as well, but can you point to to occurrences of this in RLlib?
Because I don't see any currently. All the dunder inits I find in RLlib have their args below the init.
One way on the other, we might want to standardize on one of the two.

Reply:
https://sourcegraph.com/github.com/ray-project/ray/-/blob/rllib/core/learner/learner.py?L170-206

Reply:
This is not standardized over rllib then. 🙁 
Have we decided at some point for "your" way?
I can start doing that, too.

Reply:
this might be more authoritative thread actually:

https://stackoverflow.com/questions/37019744/is-there-a-consensus-on-what-should-be-documented-in-the-class-and-init-docs

I think it says to go your way, but also do document all of the public attrs.